style.css

```diff
@@ -56,9 +56,13 @@ body {
   display: none;
 }
 
```

Comment:
- [ ] While making the changes requested by the previous code reviewer, your hamburger icon is not visible now, so I couldn't check if it is functioning. Please make sure that you work on that and also fix the changes requested previous reviewer.
![Screenshot from 2023-01-03 09-26-27](https://user-images.githubusercontent.com/92303891/210331303-458f55a3-91ce-436f-a839-e65d0cb36bea.png)